### PR TITLE
Update demo application.

### PR DESCRIPTION
**Pull request by pcostell**
* Add instructions for running.
* Remove symlink, which causes issues with our syncing client.
* Remove one-off tests which are not really good demos.

### CODE REVIEW

**Comment by pcostell** [comment]
@eddavisson -- can you take a look?

File: demo/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ndb demo application
+
+## Overview
+
+This demo shows various usages of ndb. The application is split into several
```
Comment (eddavisson) [inline demo/README.md]: s/usages of/ways to use/

File: demo/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+files handling different endpoints.
```
Comment (eddavisson) [inline demo/README.md]: maybe: consists of several different handlers

File: demo/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* hello.py: a simple hello world application.
+* guestbook.py: a simple forum application.
```
Comment (eddavisson) [inline demo/README.md]: s/forum/guestbook/

**Comment by eddavisson** [comment]
LGTM

**Pull request merged**